d2l-localize-behavior.html

```diff
@@ -47,6 +48,38 @@
 				this._observer.disconnect();
 			}
 		},
+		formatDateTime: function(val, opts) {
```

Comment:
`intl`'s API is just `format` to format a datetime, should that be consistent? (Although I think I actually like the explicit name better actually...)

Reply:
Yeah, because `intl` has separate objects you create for `new DateTimeFormat()`, `new NumberFormat()`, etc. so I needed to call those out explicitly since I'm merging those objects together here.